RFCs/0024-Blame-Collector-Options.md

```diff
@@ -0,0 +1,68 @@
+# 0024 Blame collector options
+
+## Summary
+This note outlines the proposed changes for adding more options to blame data collector.
+
+## Motivation
+To enable blame as a first class option in vstest task, we need some additional options to be added to the existing blame collector
```

Comment:
first-class option to support troubleshooting scenarios, including unattended execution in the pipeline...